pkgs/development/libraries/rapidjson/default.nix

```
@@ -1,65 +1,79 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, fetchpatch
-, pkg-config
 , cmake
+, doxygen
+, graphviz
 , gtest
 , valgrind
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "rapidjson";
-  version = "1.1.0";
+  version = "unstable-2024-04-09";
```

```suggestion
  version = "1.1.0-unstable-2024-04-09";
```
is the new convention.